tox.ini

```diff
@@ -1,24 +1,27 @@
 [tox]
 envlist =
-    py{36,37,38}-dj{22,30}
+    py{36,37,38,39}-dj{22,30,31}
     isort
     docs
 whitelist_externals=sphinx-build
 
-[travis]
-unignore_outcomes = True
+[gh-actions]
 python =
     3.6: py36,isort,docs
     3.7: py37,isort,docs
     3.8: py38,isort,docs
+    3.9: py39,isort,docs
```

Comment:
This was here before, but got my attention - we are running docs and isort checks on every Python version. I don't think we need all that here and can leave it only on python 3.9 job.

Reply:
good point, I'll change it.